install_config/revhistory_install_config.adoc

```diff
@@ -18,40 +18,38 @@
 |xref:../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Installing -> Prerequisites]
 |Clarified in the xref:../install_config/install/prerequisites.adoc#prereq-dns[DNS] section that the {product-title} 3.2 DNS changes are not automatically applied to existing clusters during an upgrade from {product-title} 3.1 to 3.2.
 
-|xref:../install_config/cluster_metrics.adoc#install-config-cluster-metrics[Enabling Cluster Metrics]
-|Added clarifying details to the xref:../install_config/cluster_metrics.adoc#metrics-using-secrets-byo-certs[Providing Your Own Certificates] section.
-
-n|xref:../install_config/upgrading/manual_upgrades.adoc#iinstall-config-upgrading-manual-upgrades[Upgrading -> Performing Manual Cluster Upgrades]
+|xref:../install_config/upgrading/manual_upgrades.adoc#iinstall-config-upgrading-manual-upgrades[Upgrading -> Performing Manual Cluster Upgrades]
 |Added an Important box about meeting prerequisites before upgrade.
 
 |xref:../install_config/upgrading/automated_upgrades.adoc#install-config-upgrading-automated-upgrades[Upgrading -> Performing Automated Cluster Upgrades]
 |Added an Important box about meeting prerequisites before upgrade.
 
-n|xref:../install_config/web_console_customization.adoc#ansible-config-web-console-customizations[Installation and Configuration -> Customizing the Web Console]
```

Comment:
@tpoitras Dropped the "Installation and Configuration ->" from a bunch of these. We only need to do that when referring to a subdirectory (like "Installing ->" or "Upgrading ->").